app/scripts/directives/logViewer.js

```diff
@@ -401,6 +401,10 @@ angular.module('openshiftConsole')
                   return;
                 }
 
+                if($scope.context.project.metadata.annotations[$filter('annotationName')('loggingVisualURL') {
+                    url = 'https://' + $scope.context.project.metadata.annotations[$filter('annotationName')('loggingVisualURL');
+                }
```

Comment:
@jcantrill It would be better to change `APIDiscovery.getLoggingURL` to take the project as a parameter and move the logic there. Then if we need the logging URL anywhere else, we don't forget to check the project annotation.

I'd suggest this code:

```js
var projectLoggingURL = $filter('annotation')('loggingVisualURL');
if (projectLoggingURL) {
  url = 'https://' + projectLoggingURL;
}